cmd/controller/app/options/options.go

```diff
@@ -21,6 +21,7 @@ import (
 	"strings"
 	"time"
 
+	"kubesphere.io/devops/pkg/apiserver/authentication/options"
```

Comment:
controller and apiserver should be independent of each other. Please avoid import the package of apiserver into the controller.

Reply:
Done. Please see https://github.com/kubesphere/ks-devops/pull/384/commits/0e88c48d491c59e25df126da6704cb649ed415c5.